dnf/cli/output.py

```diff
@@ -646,8 +639,7 @@ def listPkgs(self, lst, description, outputType, highlight_na={},
                     print("\n".join(sorted(info_set)))
 
             if thingslisted == 0:
-                return 1, [_('No packages to list')]
-            return 0, []
+                raise dnf.exceptions.Error(_('No packages to list'))
```

Comment:
This causes unit tests to fail.
It seems that output.listPkgs() is used in multiple places that do not expect the new exception raised.

Reply:
@dmach yup, I was too fast.

Pushed a new version, still running full CI stack, but it should be good now.